http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpSamDAO.scala

```diff
@@ -80,6 +80,17 @@ class HttpSamDAO[F[_]: Effect](httpClient: Client[F], config: HttpSamDaoConfig,
       )(onError)
     } yield res
 
+  def getListOfResourcePermissions(resourceId: String, resourceTypeName: ResourceTypeName, authHeader: Authorization)(
```

Comment:
I think this will also conflict with @akarukappadath's PR as she's making `SamDAO` methods take a `SamResource` instead of a `resourceId: String, resourceTypeName: ResourceTypeName`